cypress/integration/ErrorHandling/MissingAction.spec.js

```diff
@@ -10,44 +10,66 @@ import * as editDialogModal from '../../support/components/EditDialogModal'
 import * as common from '../../support/Common'
 import * as actions from '../../support/Actions'
 import * as scorerModal from '../../support/components/ScorerModal'
+import * as helpers from '../../support/Helpers'
 
-describe('ErrorHandling', () => {
-  it('Missing Action', () => {
-    models.ImportModel('z-missingAction', 'z-whatsYourName.cl')
-    modelPage.NavigateToTrainDialogs()
-    cy.WaitForTrainingStatusCompleted()
-
-    modelPage.VerifyNoErrorIconOnPage()
+describe('Missing Action - ErrorHandling', () => {
+  afterEach(helpers.SkipRemainingTestsOfSuiteIfFailed)
+  
+  context('Setup', () => {
+    it('Should import a model and wait for training to complete', () => {
+      models.ImportModel('z-missingAction', 'z-whatsYourName.cl')
+      modelPage.NavigateToTrainDialogs()
+      cy.WaitForTrainingStatusCompleted()
+    })
 
+    it('Should verify there are no error icons on the page and create a new Train Dialog', () => {
+      modelPage.VerifyNoErrorIconOnPage()
     train.CreateNewTrainDialog()
+    })
+  })
 
-    train.TypeYourMessage(common.gonnaDeleteAnAction)
-    editDialogModal.ClickScoreActionsButton()
-    train.SelectAction(common.whatsYourName)
-    
-    train.Save()
-
-    modelPage.NavigateToActions()
-    actions.DeleteAction(common.whatsYourName)
-    modelPage.NavigateToTrainDialogs()
-
-    modelPage.VerifyErrorIconForTrainDialogs()
-    train.VerifyErrorsFoundInTraining(`${common.validityIcon}${common.gonnaDeleteAnAction}`, common.gonnaDeleteAnAction, undefined)
-
-    train.EditTraining(`${common.validityIcon}${common.gonnaDeleteAnAction}`, common.gonnaDeleteAnAction, undefined)
+  context('Train', () => {
+    it('Should complete and save a simple 1 action Train Dialog', () => {
+      train.TypeYourMessage(common.gonnaDeleteAnAction)
+      editDialogModal.ClickScoreActionsButton()
+      train.SelectAction(common.whatsYourName)
```

Comment:
This actually all setup for the actual test which actually verifies what that deleting an action causes the dialog to become invalid (contain errors due to missing action).

That should be reflected in the test hierarchy.  These are just extra `it`s not really testing anything.  Would rather this all be setup.

Reply:
Yep, that sounds about right. I fixed it.